### PR TITLE
scaling for smaller screens. very rough

### PR DESCRIPTION
**Pull request by JulianGoeltz**

### CODE REVIEW

File: gui.py [hunk 1]
```diff
@@ -29,10 +36,12 @@ def __init__(self):
         self.window.Read(timeout=0.001)
         self.show_new_game()
 
-    @staticmethod
-    def create_blank_icon():
+    def create_blank_icon(self):
         buffer = BytesIO(base64.b64decode(icons.o))
         width, height = Image.open(buffer).size
+        print(width, height)
```
Comment (jakobj) [inline gui.py]: please remove print statements :P

**Comment by JulianGoeltz** [comment]
roughness:
* `scaling=1` is not the default
* `subsampling` works only in integers, i.e. `scaling` should be only `1/n`. 

reasons for existence: window does not fit on a FullHD screen. so maybe some scaling is in order, I think.
but not sure you want it to be adjustable

File: gui.py (hunk 1 above)
Comment (JulianGoeltz) [reply gui.py]: oh, well. done

**Comment by jakobj** [comment]
awesome, thanks! :+1: and merging

**Pull request merged**